pkg/fanal/analyzer/pkg/apk/apk.go

```diff
@@ -3,24 +3,27 @@ package apk
 import (
 	"bufio"
 	"context"
-	"log"
+	"fmt"
 	"os"
 	"path/filepath"
+	"sort"
 	"strings"
 
 	apkVersion "github.com/knqyf263/go-apk-version"
+	"github.com/samber/lo"
+	"golang.org/x/exp/slices"
 
 	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
 	"github.com/aquasecurity/trivy/pkg/fanal/types"
-	"github.com/aquasecurity/trivy/pkg/fanal/utils"
 	"github.com/aquasecurity/trivy/pkg/licensing"
+	"github.com/aquasecurity/trivy/pkg/log"
 )
 
 func init() {
 	analyzer.RegisterAnalyzer(&alpinePkgAnalyzer{})
 }
 
-const version = 1
+const analyzerVersion = 1
```

Comment:
Need to update the analyzer version.

```suggestion
const analyzerVersion = 2
```